CONTRIBUTING.md

```diff
@@ -30,3 +30,41 @@ Use culturally neutral language:
 
   What you may consider well-known may be entirely obscure and distracting to people from different backgrounds.
 
+### Terminology
+
+To avoid confusion around all the things called Nix, always use the following terms with capitalization as given.
+
+Terms in brackets (`[  ]`) are optional.
+Terms in parentheses (`(  )`) are abbreviations.
+
+- Nix ecosystem
```

Comment:
Nix Ecosystem.

In order to clearly mark these as tokenize d terms, they should be all Title Case.

Same for the rest...

Reply:
The "ecosystem" or "package collection" should be part of the proper name, this is why it's not capitalized.

Tangent: This is just my taste, but it think capitalization is largely pointless. Yes, it can help with readability and distinguishing proper names, but it also has a cost of abiding to fairly arbitrary rules just to keep consistency. This is just how writing systems of European descent happen to work. Many languages, [in fact most of them](https://en.m.wikipedia.org/wiki/Letter_case#Bicameral_script), don't have a notion of letter cases, and it works just fine.